### DISCUSSION TITLE
Show unread articles only

### DISCUSSION THREAD

**Discussion by scottyme** [martinrotter/rssguard#1545] [Q&A]
There is a "Show unread items only" option for Feeds. Is that option available for Articles? I want to hide all the read articles.

**Comment by martinrotter** [comment]
Yes, there is. 
![image](https://github.com/user-attachments/assets/766bc6c2-b23a-4c65-984e-1db416c572da)

**Comment by scottyme** [comment]
Aah thanks!

**Comment by scottyme** [comment]
How about the opposite--can I show the read articles only?

**Reply by martinrotter** [reply]
Just added the feature. You are welcome.

932641847

![rssguard_w2LMHL7BJX](https://github.com/user-attachments/assets/6078b91d-692a-47f6-9b22-b67efa4f3401)
